README.md

```diff
@@ -0,0 +1,56 @@
+
+You can find a complete list of all the packages for these libraries [here](./packages.md).
+
+## Getting started
+
+For your convenience, each service has a separate set of libraries that you can choose to use instead of one, large Azure package. To get started with a specific library, see the `README.md` (or `README.rst`) file located in the library's project folder.
+
+You can find service libraries in the `/sdk` directory.
+
+### Prerequisites
+The client libraries are supported on Python 2.7 and 3.5.3 or later.
+
+## Packages available
+Each service might have a number of libraries available from each of the following categories:
+* [Client - July 2019 Preview](#Client-July-2019-Preview)
+* [Client - Stable](#Client-Stable)
+* [Management](#Management)
+
+
+### Client: July 2019 Preview
+New wave of packages that we are currently releasing in **Preview**. These libraries allow you to use and consume existing resources and interact with them, for example: upload a blob. These libraries share a number of core functionalities such as: retries, logging, transport protocols, authentication protocols, etc. that can be found in the [azure-core](./sdk/core/azure-core) library. You can learn more about these libraries by reading guidelines that they follow [here](https://azuresdkspecs.z5.web.core.windows.net/PythonSpec.html).
+
+The libraries released in July preview:
+
+- [azure-cosmos](./sdk/cosmos/azure-cosmos)
+- [azure-eventhubs](./sdk/eventhub/azure-eventhubs)
+- [azure-keyvault-keys](./sdk/keyvault/azure-keyvault-keys)
+- [azure-keyvault-secrets](./sdk/keyvault/azure-keyvault-secrets)
+- [azure-identity](./sdk/identity/azure-identity)
+- [azure-storage-blob](./sdk/storage/azure-storage-blob)
+- [azure-storage-file](./sdk/storage/azure-storage-file)
+- [azure-storage-queue](./sdk/storage/azure-storage-queue)
+
+>NOTE: If you need to ensure your code is ready for production use one of the stable libraries.
+
+
+### Client: Stable
+Last stable versions of packages that have been provided for usage with Azure and are production-ready. These libraries provide you with similar functionalities to the Preview ones as they allow you to use and consume existing resources and interact with them, for example: upload a blob.
+
+### Management
+Libraries which enable you to provision specific resources. They are responsible for directly mirroring and consuming Azure service's REST endpoints. The management libraries use the `azure-mgmt-<service name>` convention for their package names.
+
+## Need help?
+* For detailed documentation visit our [Azure SDK for Python documentation](aka.ms/python-docs)
+* File an issue via [Github Issues](../../issues)
+* Check [previous questions](https://stackoverflow.com/questions/tagged/azure-sdk-python) or ask new ones on StackOverflow using `azure-sdk-python` tag.
```

Comment:
So far I was suggesting people to tag "azure" and "python", I never used the "azure-sdk-python". Do we want to mention it at least, or do we officially say we monitor only "azure-sdk-python" ?

Reply:
If we have been using "azure" and "python" then we can continue using that or at least tell people about it. I thought it'd be easier if we had a tag but I guess there is no point in introducing additional complexity

Reply:
Updated to:

```
* Check [previous questions](https://stackoverflow.com/questions/tagged/azure+python) or ask new ones on StackOverflow using `azure` and `python` tags.
```